src/mongo/db/exec/multi_plan.h

```diff
@@ -198,10 +210,14 @@ class MultiPlanStage final : public PlanStage {
     // uses -1 / kNoSuchPlan when best plan is not (yet) known
     int _bestPlanIdx;
 
-    // index into _candidates, of the backup plan for sort
+    // index into _candidates, of the backup of the plan competition
     // uses -1 / kNoSuchPlan when best plan is not (yet) known
     int _backupPlanIdx;
 
+    // Index into _candidates, of the original winner of the plan which can be used if a blocking plan .fails
```

Comment:
Use single-quotes to refer to the variable name: `Index into '_candidates', ...`

Reply:
Also, it looks like you don't need the comma here, and you should move the "." before 'fails' to be after it.